src/types.js

```diff
@@ -293,3 +294,46 @@ export type HvBehavior = {
 export type BehaviorRegistry = {
   [string]: HvBehavior,
 };
+
+// https://hyperview.org/docs/reference_behavior_attributes#trigger
+export const TRIGGERS = Object.freeze({
+  DESELECT: 'deselect',
+  LOAD: 'load',
+  LONG_PRESS: 'longPress',
+  PRESS_IN: 'pressIn',
+  PRESS_OUT: 'pressOut',
+  PRESS: 'press',
+  REFRESH: 'refresh',
+  SELECT: 'select',
+  VISIBLE: 'visible',
+});
+
+export type Trigger = $Values<typeof TRIGGERS>;
+
+export const PRESS_TRIGGERS = Object.freeze([
```

Comment:
Should we leave `freeze` on `TRIGGERS` and `PRESS_TRIGGERS`?

Reply:
I think we can revisit this in the entire project - enums should be immutable and types from enum should be marked as such (`$ReadOnly`)